### ISSUE TITLE
Icons SVGs with Border and Fill classes

### ISSUE THREAD

**Issue by shanberg** [DecipherNow/gm-ui-components#404]
# Component: Icon

## Will this modification effect the existing functionality?
Yes. New Icons may be incompatible with old icons.

## Describe your proposed change
SVG content should contain class names on separated SVG shapes. Terms such as "border" and "fill" should then be addressable in CSS to control icon styling in icon components.

Icon styling should be abstracted from the SVG element as much as possible.

Styling icons should be as simple as...
```css
svg {
   .border { stroke: #000 }
   .fill { fill: transparent }
}
```

## Supporting Images, Videos, or Links
Sample SVG exported from Sketch with shapes marked up as 'border' and 'fill'.
[cog 2.svg.zip](https://github.com/DecipherNow/gm-ui-components/files/2979734/cog.2.svg.zip)

I believe it’s a reasonable goal for us to be able to style our icons seamlessly with the rest of the app content, which should allow quicker design iteration and much greater polish in the overall app.
 
<img width="1271" alt="Screen Shot 2019-03-18 at 2 48 02 PM" src="https://user-images.githubusercontent.com/98312/54555348-e784cd00-498c-11e9-8650-b8e6c702a1cb.png">

**Comment by shanberg** [comment]
<img width="396" alt="Screen Shot 2019-03-18 at 3 06 41 PM" src="https://user-images.githubusercontent.com/98312/54556543-74308a80-498f-11e9-8012-94605f546b37.png">

I've been building new icons in this style for an update to the Data Explorer interface. So far it’s been an enormous improvement to the readability and polish in every place an icon appears.

**Comment by shanberg** [comment]
Closed by #495
